src/main/resources/jenkins/plugin/mockloadbuilder/MockLoadStep/config.jelly

```diff
@@ -1,6 +1,6 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:local="local">
-    <f:entry field="averageDuration" title="${%Average duration (seconds)}">
```

Comment:
This did not actually render the _(seconds)_ part since Stapler/Jelly misinterpreted that as an argument to pass to a localization format.